vuepress/docs/next/tutorials/README.md

```diff
@@ -7,18 +7,20 @@ Some screenshots or references to the previous version may be out-of-date and so
 only be available by building from source.
 :::
 
+
 # Learning Paths
 
-::: tip Entando simplifies the development of modern apps:
+::: tip Entando simplifies and automates the DevOps lifecycle of applications orchestrated by Kubernetes.
```

Comment:
This is a bit too narrow since the 4Cs cover more than just DevOps. Maybe "Entando simplifies the development of composable applications deployed on Kubernetes."  I actually don't love these bullets but let's keep those for now.

Reply:
I took out the blue highlighted box because it always bothered me and we don't use it anywhere else. What about the bullets needs improvement?  It feels wrong to publish when something is off on a page that will get the more traffic than most pages.